testdata/resources/config2.json

```diff
@@ -138,7 +158,7 @@
         "pool": "/cf/cf-qux-ac504dcd7f58634d",
         "ipProtocol": "tcp",
         "enabled": true,
-        "destination": "/cf/10.0.0.1:10006",
```

Comment:
this needs to be fixed port should not change

Reply:
Above comment is wrong. This port should have changed since this cycle we deleted an endpoint and added the qux endpoint. So the port of the deleted endpoint gets reassigned to the added endpoint. However, due to Dan's comment up above about unit test refactoring this config file will be returned to its previous version.